module-apps/apps-common/popups/TetheringConfirmationPopup.cpp

```diff
@@ -21,12 +21,50 @@ namespace gui
         metadata.text   = utils::translate("tethering_enable_question");
         metadata.icon   = "tethering_128px_W_G";
         metadata.action = [this]() {
-            application->bus.sendUnicast(std::make_shared<sys::TetheringEnabledResponse>(),
-                                         service::name::system_manager);
-            app::manager::Controller::sendAction(application, app::manager::actions::Home);
+            // check if there is any unsaved data in any window in stack
+            bool mayUnsavedData  = false;
```

Comment:
may doesn't sound like true or false 
If this variable doesn't keep the probability or % chance that some data is unsaved consider rename it to 
IsDataUnsaved